sensor/common/imageutil/util.go

```diff
@@ -5,13 +5,24 @@ import (
 	"github.com/stackrox/rox/sensor/common/registry"
 )
 
+// IsInternalImageOptions provides options for determining if an image is "internal".
+type IsInternalImageOptions struct {
+	RegistryStore *registry.Store
+}
+
 // IsInternalImage determines if the image represented by the given name
 // is an "internal" image. An internal image is one which is hosted by an internal registry.
 // An internal registry is on which is only accessible from within the cluster in which it lives.
-func IsInternalImage(image *storage.ImageName) bool {
+func IsInternalImage(image *storage.ImageName, opts *IsInternalImageOptions) bool {
```

Comment:
Just pass the registry into this function? It's not clear why we would want to use an options pattern here

Reply:
I was originally trying to avoid passing `nil` to use the default, but that's exactly what I'm doing here, so I'll just pass the registry

Reply:
It was also meant to somewhat hide the registry from the users